Pod/Classes/NSURLSession/Matchers/SimpleRequestMatcher.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SimpleRequestMatcher.swift
+//  Pods
+//
+//  Created by Sam Dean on 02/02/2016.
+//
+//
+
+import Foundation
+
+/**
+ This struct matches by matching the path to a regular expression and by 
+ comparing the methods for equality.
+*/
+struct SimpleRequestMatcher : RequestMatcher {
+    
+    let pathMatcher: NSRegularExpression
+    
+    let method: String
+    
+    init(url: NSURL, method: String) {
+        let path = NSRegularExpression.escapedPatternForString(url.absoluteString)
+        try! self.init(expression: "^\(path)$", method: method)
+    }
+    
+    init(expression: String, method: String) throws {
+        try self.pathMatcher = NSRegularExpression(pattern: expression, options: NSRegularExpressionOptions.AnchorsMatchLines)
+        self.method = method
+    }
+    
+    func matches(request: NSURLRequest) -> Bool {
+        guard request.HTTPMethod == self.method else { return false }
+        
+        let path = request.URL?.absoluteString ?? ""
+        let options = NSMatchingOptions(rawValue: 0)
+        let range = NSMakeRange(0, (path as NSString).length)
```

Comment:
We have [precedents for this in our projects](https://github.com/net-a-porter-mobile/XCTest-Gherkin/blob/master/Pod/Core/XCTestCase%2BGherkin.swift#L216) as well as [other open source projects](https://github.com/fastlane/snapshot/blob/14247491a792016f83ac7aa33effa77a3b416d31/lib/assets/SnapshotHelper.swift#L54). I think it should be fine.